src/devtools/client/inspector/inspector.ts

```diff
@@ -81,7 +81,7 @@ export class Inspector {
   panelDoc: Document | null;
   panelWin: Window | null;
   store: UIStore | null;
-  highlighter: any;
+  highlighter: typeof Highlighter;
```

Comment:
Nice, I like this direction for shimming in the older stuff much better than `any`.